crates/rmf_site_editor/src/widgets/inspector/mod.rs

```diff
@@ -350,53 +358,81 @@ impl<'w, 's> WidgetSystem<Tile> for Inspector<'w, 's> {
                     return;
                 };
 
-                let Some(mut selection) = selection.0 else {
-                    ui.label("Nothing selected");
+                if selection.selected.is_empty() {
+                    ui.label("Nothing selected or multiple models selected!");
```

Comment:
I'm confused by the addition of "or multiple models selected". Is this a leftover from an earlier implementation?